### ISSUE TITLE
Reconsider Default Exclusions / Cloud Files Issue

### ISSUE THREAD

**Issue by Medowar** [windirstat/windirstat#56]
Hi, 
I just tested out the Beta-Client, after reading your post on reddit. It is significantly faster than the old client, however it seems to lack in detecting some files.
According to the harddrive properties, 106GB are taken on my drive. 
New Client reports 61,8GB(logical 63,1GB) of files.
Old Client is much more in the right ballpark with 100,1GB.
![2024-05-11 00_19_12-Einstellungen](https://github.com/windirstat/windirstat/assets/1496389/93dc2fa3-874f-4ff5-b3f1-dcb09d972ce9)
System: Win11x64, current Updates. SSD is basic NTFS formatted, System originally installed Win10, then upgraded.
Both clients in default config, right after downloading. 

One of the main differences seems to be the detection of used space of virtual files. I have both nextcloud and onedrive with virtual files enabled on the device, the old client does detect the actual storage usage, whereas the new one only reports 0B. But there is some of these files downloaded locally and kept as offline-cache.
Recycle.bin also seems to be missing from the view.

If helpful, I can provide you with a full system report in private.
Thanks for reviving this beloved but long inactive tool :)
Regards.

**Comment by NoMoreFood** [comment]
@Medowar Can you see if deselecting this option gives you closer to the correct result?

![image](https://github.com/windirstat/windirstat/assets/12881023/616cae40-666f-427c-8d6d-2c13a1442425)

**Comment by Medowar** [comment]
Delecting skipping protected items gives a value very close to the actual usage, even closer than the old client. But I think, that this option should default to off, since when reclaiming space, I want so see everything by default, even Windows. Some issues would probably not be found that way, like exploding cbs.log file or huge temp dir.

Onedrive and Nextcloud are also both shown as 0B in both physical and logical size.
I tried playing around with different settings, but none changed anything.

**Comment by NoMoreFood** [comment]
@Medowar I'll reconsider not showing hidden and protected by default.  I don't see that behavior with OneDrive. I'll try installing Nextcloud and see if I can reproduce that.

**Comment by atsju** [comment]
Follow-up of my test in https://github.com/windirstat/windirstat/issues/42#issuecomment-2144582718
I applied the fixed proposed here but Windirstats found only 384GB out of 469GB.

**Comment by NoMoreFood** [comment]
@atsju How does that compare to the previous version of WinDirStat?  Did you try running the application "As Administrator"? If the previous version found more, can you line them up side-to-side and see where the largest directory is where the new version is missing files?

**Comment by atsju** [comment]
I cannot run de application "as administrator" because of company security policy.
![image](https://github.com/windirstat/windirstat/assets/4628382/6807aae8-2aea-4326-af0a-658757c53752)
![image](https://github.com/windirstat/windirstat/assets/4628382/33dfd51b-4265-45d5-bc1b-b349fea71b5c)

I tracked it down to SharePoint and oneDrive files. Windows stats gives about 10.9GB in sharepoint + 15.0GB in oneDrive that are not seen at all by new tool (old tool saw 72.6GB + 14.9GB).  
This is all about the difference between old and new version. Appart from these 2 lines, results are consistent.  
Let's ignore oneDrive and Sharepoint. Windows totals to 426-10.9-15=400GB.   
Old version gives 411-72.6-14.9=323.5GB
New version gives 328GB.  
Consistent but wrong. Both are missing ~70GB

Tools missed 3GB in C:\Windows but I don't know where the 70GB are from.

I did an additional test and hit "properties" on all files&folders in C:
![image](https://github.com/windirstat/windirstat/assets/4628382/a47397cc-d53a-41af-b9cd-3d1c3db59d0a)
Got 340GB. Which is consistent with WindirStats when you take into account missing sharepoint and oneDrive on windirstats and missing hiberfil.sys+pagefil.sys (17.4GB) on windows side.

To summarize: 

- I have a discrepancy inside windows where 70GB are lost in the void. Do you have any idea ?    
- Old version was counting files in Sharepoint that were supposed to be ignored.   
- New version simply ignores all Sharepoint files.

**Comment by NoMoreFood** [comment]
Can you see if this version behaves any differently? I made some adjustments in this area.

https://github.com/windirstat/windirstat-next/releases

**Comment by atsju** [comment]
as of today:  
properties: 432GB  
![image](https://github.com/windirstat/windirstat/assets/4628382/328e9d65-b445-4d30-9766-a66b394880fb)
properties on all files and folders: 362GB (missing pagefile.sys 4.7GB and hiberfil.sys 12.7GB) => total **379.4GB**
windirStats 9.0.0-9a53:  **379.8GB**  
![image](https://github.com/windirstat/windirstat/assets/4628382/fac6a4bf-d26b-4eaf-92de-237e1fb9f48f)  

onedrive: 15.3GB  
sharepoint: 11.3GB  

Both oneDrive and Sharepoint are correctly tacken into account and total is correct. This is good 👍 

I don't know where the internal windows discrepancy comes from but I tend to say version 2.0.0.0 9a53 is correct 💯
